core/src/avm1/globals/context_menu.rs

```diff
@@ -0,0 +1,154 @@
+use crate::avm1::activation::Activation;
+use crate::avm1::error::Error;
+use crate::avm1::object::TObject;
+use crate::avm1::property::Attribute;
+use crate::avm1::Object;
+use crate::avm1::{ScriptObject, Value};
+use crate::context::UpdateContext;
+use gc_arena::MutationContext;
+
+pub fn constructor<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    context: &mut UpdateContext<'_, 'gc, '_>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let mut so = this.as_script_object().unwrap();
+
+    let callback = args
+        .get(0)
+        .unwrap_or(&Value::Undefined)
+        .coerce_to_object(activation, context);
+
+    so.set("onSelect", callback.into(), activation, context)?;
+
+    so.force_set_function(
+        "copy",
+        copy,
+        context.gc_context,
+        Attribute::DontEnum,
+        Some(context.system_prototypes.function),
+    );
+
+    so.force_set_function(
+        "hideBuiltInItems",
+        hide_builtin_items,
+        context.gc_context,
+        Attribute::DontEnum,
```

Comment:
These should be set on the prototype in `create_proto`:
`trace(ContextMenu.prototype.hideBuiltInItems); // [Function]`
Looks like these are Attribute::DontDelete too